src/scss/ui/_tag.scss

```diff
@@ -0,0 +1,68 @@
+.tag-container {
+    display: flex;
+    flex-wrap: wrap;
+
+    .tag {
+        background-color: $tag-bg-color;
+        height: 23px;
+        border: 1px solid gray;
+        padding: 0 .2rem;
+        margin: 0 1rem;
```

Comment:
single tag shouldnt have margins. Margins should be made by container (`tags-list` etc)